app/layout.tsx

```diff
@@ -28,7 +28,7 @@ export default function RootLayout({
         <link rel="alternate icon" href="/SerotrackerLogo.png" />{" "}
         {/* Fallback for older browsers */}
       </head>
-      <body className={clsx(inter.className, "text-black no-scrollbar")}>
+      <body className={clsx(inter.className, "text-black no-scrollbar overflow-hidden")}>
```

Comment:
Commenting on a random line, you can also download csvs on the table with WHO regions and age groups. I'm assuming that one's fine?